tiny_dnn/util/parameter_init.h

```diff
@@ -53,10 +53,10 @@ class constant : public scalable {
   constant() : scalable(float_t{0}) {}
   explicit constant(float_t value) : scalable(value) {}
 
-  void fill(parameter *p, size_t fan_in, size_t fan_out) override {
+  void fill(Parameter *parameter, size_t fan_in, size_t fan_out) override {
```

Comment:
@beru that's a clear example why I'm a fan of camelCase 😄 I can understand that snake_case is attractive at user level but I don't think it's practic. I don't want to start a forever discussion.